### ISSUE TITLE
Verify Callback Events

### ISSUE THREAD

**Issue by bradshjg** [novafloss/django-docusign#77]
I might be missing something, but is there anything currently implemented that verifies callback (POST) events actually originate from Docusign?

**Comment by zebuline** [comment]
@bradshjg Yes, but it is deprecated since the 2.0 version: https://github.com/novafloss/django-docusign/blob/master/CHANGELOG#L23, because of many issues with callbacks.
